aii-opennebula/src/main/perl/opennebula.pod

```diff
@@ -96,8 +99,6 @@ All these paths are relative to /system/aii/hooks/remove
 
 Stop running VM
 
-=over 4
```

Comment:
if you remove the ```=over``` here, you'll have to remove one of the  ```=back``` a bit further as well.